lib/instrumentation/winston.js

```diff
@@ -12,16 +12,20 @@ module.exports = function instrument(agent, winston, _, shim) {
   const config = agent.config
   if (
     !(
-      config.application_logging &&
       config.application_logging.enabled &&
-      config.application_logging.local_decorating &&
-      config.application_logging.local_decorating.enabled
+      (config.application_logging.forwarding.enabled ||
+        config.application_logging.metrics.enabled ||
+        config.application_logging.local_decorating.enabled)
     )
   ) {
+    winston.__NR_instrumented = false
```

Comment:
this will get overwritten by `lib/shimmer`. it sets it to true since we've registered instrumentation.  I think you're thinking of the right kind of thing.  I'm thinking though we may want to add something to the api that can identify it's coming from agent vs log enricher

Reply:
we might be able to check for the existence of `config.application_logging && config.application_logging.enabled === false`